test/util/util.go

```diff
@@ -345,7 +345,9 @@ func AssertServiceInstanceCondition(t *testing.T, instance *v1beta1.ServiceInsta
 	}
 
 	if !foundCondition {
-		t.Fatalf("%v condition not found", conditionType)
+		if status != v1beta1.ConditionFalse || len(reason) != 0 {
```

Comment:
how weird, this function wasn't used before this. and the other one was used only once.

change makes sense, but maybe could use a comment that we have indeed found a condition because we have a set status

Reply:
I'll create a separate function for the false-or-absent assert.

Reply:
ok

Reply:
I meant to remove this in lieu of the new function. I've pushed up a new commit with this removed.